toolkit/tools/pkg/rpmssnapshot/rpmssnapshot.go

```diff
@@ -53,42 +52,38 @@ const (
 	rpmSpecBuiltRPMRegexDistributionIndex
 	rpmSpecBuiltRPMRegexArchitectureIndex
 	rpmSpecBuiltRPMRegexMatchesCount
+
+	chrootName = "rpmssnapshot_chroot"
+	runChecks  = false
 )
 
 type SnapshotGenerator struct {
-	chroot        *safechroot.Chroot
-	buildDirPath  string
-	workerTarPath string
-}
-
-// New creates an unitialized RPMs snapshot generator.
-func New(buildDirPath, workerTarPath string) *SnapshotGenerator {
-	return &SnapshotGenerator{
-		buildDirPath:  buildDirPath,
-		workerTarPath: workerTarPath,
-	}
+	simpleToolChroot simpletoolchroot.SimpleToolChroot
 }
 
-// GenerateSnapshot generates a snapshot of all packages built from the specs inside the input directory.
-func (s *SnapshotGenerator) GenerateSnapshot(specsDirPath, outputFilePath, distTag string) (err error) {
-	err = s.initializeChroot(specsDirPath)
-	if err != nil {
-		return
-	}
-	defer s.cleanUp()
+// New creates a new snapshot generator.
```

Comment:
Maybe let's also explicitly mention that `CleanUp` must be called if this returns successfully.